plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -184,4 +186,43 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    private static IcebergColumnHandle pathColumnHandle()
+    {
+        IcebergColumnHandle icebergColumnHandle = new IcebergColumnHandle(
+                columIdentity(FILE_PATH),
+                FILE_PATH.type(),
+                ImmutableList.of(),
+                FILE_PATH.type(),
+                Optional.empty());
+
+        return icebergColumnHandle;
+    }
+
+    public static ColumnMetadata pathColumnMetadata()
+    {
+        return ColumnMetadata.builder()
+                .setName(FILE_PATH.columnName())
+                .setType(FILE_PATH.type())
+                .setHidden(true)
+                .build();
+    }
+
+    public static List<IcebergColumnHandle> withPathColumnHandle(List<IcebergColumnHandle> columnHandles)
```

Comment:
We don't really need this extra method in IcebergColumnHandle.
It is used only once in IcebergMetadata anyway.